docs/zh-cn/keycodes_basic.md

```diff
@@ -0,0 +1,256 @@
+| `KC_APPLICATION`     | `KC_APP`                       | Application键 (Windows 中对应 Menu 键)    |
+| `KC_KB_POWER`        |                                | 系统电源键                                |
+| `KC_EXECUTE`         | `KC_EXEC`                      | Execute键                                 |
+| `KC_HELP`            |                                | Help键                                    |
+| `KC_MENU`            |                                | Menu键                                    |
+| `KC_SELECT`          | `KC_SLCT`                      | Select键                                  |
+| `KC_STOP`            |                                | Stop键                                    |
+| `KC_AGAIN`           | `KC_AGIN`                      | Again键                                   |
+| `KC_UNDO`            |                                | 撤销键                                    |
+| `KC_CUT`             |                                | 剪切键                                    |
+| `KC_COPY`            |                                | 复制键                                    |
+| `KC_PASTE`           | `KC_PSTE`                      | 粘贴键                                    |
+| `KC_FIND`            |                                | Find键                                    |
+| `KC_KB_MUTE`         |                                | Mute键                                    |
+| `KC_KB_VOLUME_UP`    |                                | 音量+                                     |
+| `KC_KB_VOLUME_DOWN`  |                                | 音量-                                     |
+| `KC_ALTERNATE_ERASE` | `KC_ERAS`                      | Alternate Erase键                         |
+| `KC_SYSTEM_REQUEST`  | `KC_SYRQ`                      | SysReq/Attention键                        |
+| `KC_CANCEL`          | `KC_CNCL`                      | Cancel键                                  |
+| `KC_CLEAR`           | `KC_CLR`                       | Clear键                                   |
+| `KC_PRIOR`           | `KC_PRIR`                      | Prior键                                   |
+| `KC_RETURN`          | `KC_RETN`                      | Return键                                  |
+| `KC_SEPARATOR`       | `KC_SEPR`                      | Separator键                               |
+| `KC_OUT`             |                                | Out键                                     |
+| `KC_OPER`            |                                | Oper键                                    |
+| `KC_CLEAR_AGAIN`     | `KC_CLAG`                      | Clear/Again键                             |
+| `KC_CRSEL`           | `KC_CRSL`                      | CrSel/Props键                             |
+| `KC_EXSEL`           | `KC_EXSL`                      | ExSel键                                   |
+
+## 媒体键
+
+这些键码不在**键盘/小型键盘使用手册**的*使用（Usage）*章节中。`SYSTEM_` 前缀的键码在*通用台式机（Generic Desktop）*章节，其余部分在*用户（Consumer）*章节。
```

Comment:
I'm unsure on this translation, I'd like to see some input from others if this is okay.